testing/test_warnings.py

```diff
@@ -106,3 +109,29 @@ def test_ignore(testdir, pyfile_with_warnings, method):
     ])
     assert WARNINGS_SUMMARY_HEADER not in result.stdout.str()
 
+
+
+def test_unicode(testdir, pyfile_with_warnings):
+    testdir.makepyfile('''
```

Comment:
Sorry, I don't know which section should I add the entry to CHANGELOG, could you please tell me that?